### DISCUSSION TITLE
Request for Information About the OMAP File Structure

### DISCUSSION THREAD

**Discussion by glucawork** [OpenOrienteering/mapper#2302] [Questions & Answers]
Hi everyone!

I'm developing a QGIS plugin to import maps in the OMAP format. To proceed, I would greatly appreciate some assistance in understanding the XML structure in detail. Could anyone kindly help me with this?

Thank you in advance for your help! 😊

- gluca

**Comment by dg0yt** [comment]
Apart from the verbosity of the XML format, there is not much documentation unfortunately.
These links are related but probably not adding much value:
https://github.com/OpenOrienteering/mapper/issues/182
https://www.openorienteering.org/api-docs/mapper/file_format.html

**Comment by pkturner** [comment]
I'm willing to answer any questions you have about the georeferencing section of the XML, and how the map's georeferencing interprets the coordinates of objects.
